src/styles/Error.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+export const ErrorMessage = styled.h1`
```

Comment:
I thought we discussed making this the base text size (which should be 16px)? It's gigantic as an h1 which makes it difficult to show long errors. Additionally, we talked about truncating it with an ellipsis.